### DISCUSSION TITLE
New development model, maybe?

### DISCUSSION THREAD

**Discussion by ryjones** [hyperledger/fabric-docs-i18n#384] [Show and tell]
I would like to propose a change to documentation development: the use of short-lived branches on the main repo.

I've added a rule for branches named `WIP/*` that allows force pushes and the like, as well as auto delete on merge.

The way this makes it easier is your branch for your branch is always fairly up-to-date with the main `release-2.2` branch.

Thoughts? This is how #381 worked.

**Comment by aakzubaidi** [comment]
Thanks for letting us know.

**Reply by ryjones** [reply]
This is my idea: https://youtu.be/QJTj9QQCdG8

**Reply by aakzubaidi** [reply]
Thanks. This really helps.

**Comment by stone-ch** [comment]
I think this is a better way for contributing translations. But it is more complex, maybe we should add more introductions or tutorials.
